### PR TITLE
feat: add moko mvvm-compose

### PR DESCRIPTION
**Pull request by marzelwidmer**
Based on this issue : https://github.com/terrakok/Compose-Multiplatform-Wizard/issues/23

### CODE REVIEW

File: src/jsMain/kotlin/ui/Content.kt [hunk 1]
```diff
@@ -159,6 +160,7 @@ val Content = FC<AppProps> { props ->
                     }
 
                     val deps = setOf(
+                        DependencyBox(MokoMvvm, false),
```
Comment (terrakok) [inline src/jsMain/kotlin/ui/Content.kt:163]: Move it after the KotlinxCoroutinesCore, please (first elements are selected by default)

File: src/commonMain/kotlin/wizard/Dependencies.kt [hunk 2]
```diff
@@ -308,3 +308,14 @@ val BuildKonfigPlugin = Dependency(
     catalogName = "buildKonfig",
     platforms = emptySet()
 )
+val MokoMvvm = Dependency(
```
Comment (terrakok) [inline src/commonMain/kotlin/wizard/Dependencies.kt:311]: You are supposed to add it to the tests and check that all tests are green. (to `allDependencies` at least)

**Review by terrakok — CHANGES_REQUESTED** [review]

File: src/jvmTest/kotlin/wizard/GeneratorTest.kt [hunk 3]
```diff
@@ -281,6 +283,7 @@ class GeneratorTest {
                 ktor-client-js = { module = "io.ktor:ktor-client-js", version.ref = "ktor" }
                 kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version.ref = "kotlinx-coroutines" }
                 kotlinx-coroutines-android = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-android", version.ref = "kotlinx-coroutines" }
+                moko-mvvm = { module = "dev.icerock.moko:mvvm-compose", version.ref = "moko-mvvm]" }
```
Comment (terrakok) [inline src/jvmTest/kotlin/wizard/GeneratorTest.kt:286]: ]

**Comment by terrakok** [comment]
Before a push check tests locally to avoid small problems

File: src/jvmTest/kotlin/wizard/GeneratorTest.kt (hunk 3 above)
Comment (marzelwidmer) [reply src/jvmTest/kotlin/wizard/GeneratorTest.kt:286]: Thanks :)

**Comment by marzelwidmer** [comment]
> Before a push check tests locally to avoid small problems

`BUILD SUCCESSFUL in 3m 3s
19 actionable tasks: 4 executed, 15 up-to-date`

rebase and push again :)

**Comment by TheDome0** [comment]
just to clarify as I was about to look into this too: this PR is about [moko-mvvm](https://github.com/icerockdev/moko-mvvm), not actually [moko-resources](https://github.com/icerockdev/moko-resources) like #23 
Or do you want to add it too

**Comment by marzelwidmer** [comment]
yes you are right. my fault. I think should be a separate PR then.

**Pull request merged**